axelor-production/src/main/java/com/axelor/apps/production/service/manuforder/ManufOrderServiceImpl.java

```diff
@@ -108,6 +110,7 @@ public class ManufOrderServiceImpl implements ManufOrderService {
   protected ProdProductRepository prodProductRepo;
   protected ProductCompanyService productCompanyService;
   protected BarcodeGeneratorService barcodeGeneratorService;
+  protected ProductStockLocationServiceImpl productStockLocationServiceImpl;
```

Comment:
Please inject the interface instead of injecting the implementation (you will need to add the method to the interface, see comment below).